docs/experimental_features.md

```diff
@@ -1,30 +1,27 @@
-# Experimental Features
+# Experimental features
 
 !!! warning
-    These are experimental features of rattler-build and may change or go away completely.
+    These are experimental features of `rattler-build` and may change or go away completely.
 
 
-Currently only `build` & `rebuild` command supports the following experimental features.
+Currently only the `build` and `rebuild` commands support the following experimental features.
 
-To enable them use the `--experimental` flag with the command.
+To enable them, use the `--experimental` flag with the command.
 Or, use the environment variable, `RATTLER_BUILD_EXPERIMENTAL=1`.
 
-Jinja functions
----------------
+## Jinja functions
 
 ### `load_from_file(<file_path>)`
 
-The jinja function `load_from_file` allows loading from files, specifically, it allows loading from `toml`, `json`
-and `yaml` to an object to allow to fetch things directly from it.
-While it loads all other files as strings.
+The Jinja function `load_from_file` allows loading from files; specifically, it allows loading from `toml`, `json`,
+and `yaml` file types to an object to allow it to fetch things directly from the file.
+It loads all other files as strings.
 
 #### Usage
 
-This is useful when you have the project description in a well defined project file, such as, `Cargo.toml`, `package.json`, `pyproject.toml`, `package.yaml`, or `stack.yaml`. And would like to keep the recipe as simple as possible, while not worrying about keeping changes in sync, perhaps using it with CI/CD.
+`load_from_file` is useful when there is a project description in a well-defined project file such as `Cargo.toml`, `package.json`, `pyproject.toml`, `package.yaml`, or `stack.yaml`. It enables the recipe to be preserved in as simple a state as possible, especially when there is no need to keep the changes in sync; some example use cases for this are with CI/CD infrastructure or when there is a well-defined output format.
```

Comment:
This paragraph needs double-checking; I want to confirm that the edited wording is still true to the original meaning.